### PR TITLE
Add a navbar

### PR DESCRIPTION
**Pull request by dfaulken**
Links to '#' are for things that don't exist yet.

Addresses #15, but does not close.

screen shot 2017-08-17 at 4 47 34 pm

### CODE REVIEW

**Comment by Anbranin** [comment]
Could you highlight the one of the page you're on so we know where we are

**Comment by dfaulken** [comment]
Yes! ab805ab

screen shot 2017-08-17 at 5 06 57 pm

Let me know how you want the "current" link to be styled and I'll make that change.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def navbar_link(text, url)
+    button_classes = %w[button btn btn-default navbar-btn navbar-left]
+    button_classes << 'current' if current_page? url
```
Comment (jforgue) [inline app/helpers/application_helper.rb]: Not quite as fun as my solution! `current_page?(url) ? 'current' : ''` But I suppose it is easier to understand.

**Comment by Anbranin** [comment]
Not yellow

**Comment by Anbranin** [comment]
Green to match "showing all passengers" lol

**Comment by dfaulken** [comment]
Is that a serious request? Can't tell.

**Comment by Anbranin** [comment]
I regret not putting my foot down from the very beginning and telling them to change that color. I think everything should be shades of grey and maroon except the passenger expiring colors.

**Comment by Anbranin** [comment]
This is serious: What about that purple color that the table is?

**Comment by dfaulken** [comment]
I don't think that passes WCAG with that grey-ish text

File: app/views/layouts/_navbar.haml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+%nav.navbar.navbar-default
+  .container-fluid
+    = navbar_link 'All Passengers', passengers_url
+    = navbar_link 'Add New Passenger', new_passenger_url
+    = navbar_link 'Dispatch Log', '#'
+    =link_to '#' do
```
Comment (Anbranin) [inline app/views/layouts/_navbar.haml]: Do we need a `destroy` action in the `sessions_controller`?

Comment (dfaulken) [inline app/views/layouts/_navbar.haml]: This is a placeholder for https://github.com/umts/st-pax/issues/15

**Comment by dfaulken** [comment]
I'm just going to merge and we can bikeshed about colors later.

**Pull request merged**